app/assets/javascripts/modules/help.js.coffee

```diff
@@ -0,0 +1,253 @@
+                    <small>Do not show again</small>
+                  </label>
+                </div>
+              </div>"
+
+  tour = [{
+      title: "Search"
+      content: '<p>Use Earthdata Search Client\'s natural language processing-enabled search tool to quickly narrow
+                down to relevant collections.  An example search phrase could be "Land Surface Temperature over
+                Texas last month".  Results will be displayed in the collection panel below.</p>
+                <p>If you would prefer to pick a temporal range from a calendar (<i class="fa fa-clock-o"></i>),
+                draw spatial boundaries (<i class="fa fa-fw fa-crop"></i>), or
+                upload a shapefile (<i class="fa fa-fw fa-crop"></i> <i class="fa fa-arrow-right"></i> <i class="fa fa-file-o"></i>),
+                use the buttons to the right of the search box.</p>
+                <p>To start your search session over, click the eraser icon (<i class="fa fa-eraser"></i>) to clear all of your set filters.</p>
+                '
+      element: '#keywords'
+      placement: 'bottom'
+      showNext: true
+      },{
+      title: "Search Results"
+      content: '<p>Search results will be shown in the Matching Collections panel below.  Each result will have summary 
+                information along with relevant badges to allow you to quickly scan your search results to find the
+                right collection for you.  The panel can be resized by clicking and dragging the bar above the "Matching
+                Collections" tab.</p>
+                <p>To view more information about a collection, click on the <i class="fa fa-info-circle"></i> icon.</p>
+                <p>To view granules available for download, click anywhere on a collection.</p>
+                <p>Click on the <i class="fa fa-plus"></i> icon to add a collection to a project, which allows you to compare multiple collections.</p>
+'
+      element: '#collection-results-list'
+      placement: 'top'
+      showNext: true
+      },{
+      title: "Facets"
+      content: "<p>Refine your search further with available facets, such as:
+                <div style='margin-left: 15px;'><ul style='list-style-type: disc;'>
+                  <li>Features</li>
+                  <li>Keywords</li>
+                  <li>Platforms</li>
+                  <li>Instruments</li>
+                  <li>Organizations</li>
+                  <li>Projects</li>
+                  <li>Processing Levels</li>
+                </ul></div></p>"
+      element: "#master-overlay-parent"
+      placement: 'right'
+      showNext: true
+      top: null
+    }, {
+      title: "Map Tools"
+      content: '<p>Use these standard map tools to configure and position the map as well as enable certain spatial search tools.</p>'
+      placement: 'left'
+      element: '.leaflet-control-zoom-in'
+      showNext: true
+    }, {
+      title: 'Toolbar'
```

Comment:
On this last stop of the tour, it would be nice to turn the "End Tour" button blue as that is the primary call-to-action at that point.